src/quo2/screens/avatars/account_avatar.cljs

```diff
@@ -0,0 +1,48 @@
+(ns quo2.screens.avatars.account-avatar
+  (:require [reagent.core :as reagent]
+            [quo.react-native :as rn]
+            [quo.previews.preview :as preview]
+            [quo.design-system.colors :as colors]
+            [quo2.components.avatars.account-avatar :as quo2]))
+
+(def descriptor [{:label   "Icon"
+                  :key     :icon
+                  :type    :select
+                  :options [{:key   :main-icons/wallet
+                             :value "Wallet"}
+                            {:key   :main-icons/token
+                             :value "Token"}
+                            {:key   :main-icons/status
+                             :value "Status"}]}
+                 {:label   "Size"
+                  :key     :size
+                  :type    :select
+                  :options [{:key   20
+                             :value "Small"}
+                            {:key   24
+                             :value "Medium"}
+                            {:key   32
+                             :value "Big"}
+                            {:key   48
+                             :value "Very big"}
+                            {:key   80
+                             :value "Seriously Big!"}]}])
```

Comment:
I am not sure how these options will be informative to the tester/reviewer. But still thank you for removing unused changes